grapher/axis/AxisConfigInterface.ts

```diff
@@ -8,4 +8,40 @@ export interface AxisConfigInterface {
     max?: number
     canChangeScaleType?: boolean
     removePointsOutsideDomain?: boolean
+    hideAxis?: boolean
+
+    /**
+     * Whether the axis domain should be the same across faceted charts (if possible)
+     */
+    facetAxisRange?: FacetAxisRange
```

Comment:
Yes it's domain. I was thinking whether I should rename it (came from an earlier PR) because it potentially changes charts already in the database. But given none of them should be public yet, I should probably go ahead and rename it before it's too late.

Reply:
Yep that was my thinking too. My guess is there shouldn't be any chart out there where this is already serialized, so still easy to do a rename.